run.sh

```diff
@@ -85,6 +85,24 @@ function setup() {
       run_scripts $BINDER "create.sh"
     popd
     run_scripts "redis" "create.sh"
+    if [ "$PLATFORM" == "cloudfoundry" ];
+    then
+    export SPRING_PROFILES_ACTIVE=cloud1
+    run_scripts "server" "create.sh"
+    SERVER_URI=$(cf app scdf-server | grep dataflow-server- | awk '{print $2}' | sed 's:,::g')
+    SERVER_URI="http://$SERVER_URI"
+    wget $SERVER_URI/features -O features.txt
+    if grep -Fxq "{\"analyticsEnabled\":true,\"streamsEnabled\":false,\"tasksEnabled\":true}" features.txt
+        then
+        echo "Spring Cloud Config server properties are updated correctly."
+        rm features.txt
+        else
+        echo "Spring Cloud Config server properties are not available for the SCDF server. Tests fails"
+        exit 0
+    fi
```

Comment:
Not very aware of what good practices are in terms of bash coding, but isn't there something odd with indentation here?